src/Servers/IIS/AspNetCoreModuleV2/InProcessRequestHandler/dllmain.cpp

```diff
@@ -128,7 +129,7 @@ CreateApplication(
             std::unique_ptr<InProcessOptions> options;
             THROW_IF_FAILED(InProcessOptions::Create(*pServer, pSite, *pHttpApplication, options));
             // Set the currently running application to a fake application that returns startup exceptions.
-            auto pErrorApplication = std::make_unique<StartupExceptionApplication>(*pServer, *pHttpApplication, g_hServerModule, options->QueryDisableStartUpErrorPage(), hr);
+            auto pErrorApplication = std::make_unique<StartupExceptionApplication>(*pServer, *pHttpApplication, g_hServerModule, options->QueryDisableStartUpErrorPage(), hr, g_errorPageContent);
```

Comment:
If `g_errorPageContent` is only ever used once, and only right here, then we can `std::move` it all the way to where it's assigned and avoid any copies.